bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+
+import os
+import sys
+import subprocess
+import argparse
+import json
+
+# TODO
+# We are using external apps just for the moment.
+# These will be removed for native libraries.
+AWSCLIENT = "/usr/local/bin/aws"
+PGCLIENT = "/usr/bin/psql"
+
+# NZSL: Is there a database url defined in the environment?
+DATABASE_URL = os.getenv("DATABASE_URL", None)
+if not DATABASE_URL:
+    print("You must define DATABASE_URL in the environment.", file=sys.stderr)
+    exit()
```

Comment:
issue: I don't think we should be explicitly checking for this or assuming it will be set

Like the AWS CLI, Postgres [supports environment variables for configuring the connection](https://www.postgresql.org/docs/current/libpq-envars.html) - while `DATABASE_URL` isn't one of them, I think it should be safe for us to pass that value to `psql` since it should either be not set (in which case the value will be empty and thus ignored) or set to a valid connection string.

In the case that it's empty, that would mean you could alternatively configure the credentials using the native pg env variables.

Reply:
Currently DATABASE_URL is the only means of connecting (inside the app - see lines 174-181).
Let's revisit this one later?

Reply:
Hmm, though that said, that variable will just result in a null string if it's not occupied, which would I _presume_ cause the `psql` client to fall back on other means.  Not sure.  Needs revisiting.

Reply:
Yes, please re-read my initial comment again as it already covers this

Reply:
Done.